Task_30.1

```diff
@@ -0,0 +1,19 @@
+ALTER TABLE new_students
+RENAME to old_students;
+
+ALTER TABLE old_students
+ADD rating integer;
+
+INSERT into old_students(id, first_name, second_name, age, group_id, rating)
+VALUES (4, 'Carl', 'Johnson', 26, 2, 4);
+
+INSERT into old_students(id, first_name, second_name, age, group_id, rating)
+VALUES (5, 'Peter', 'Jackson', 28, 2, 5);

```

Comment:
You could have combined these two inserts. Of course, the way you wrote it is till 100% valid